jose-jwt/JWTSettings.cs

```diff
@@ -13,7 +13,7 @@ public class JwtSettings
     {
         public JwtSettings()
         {
-#if NET472 || NETSTANDARD2_1 
+#if NET472 || NETSTANDARD2_1 || NET

```

Comment:
shouldn't it be `NET8_0` ? What `NET` specifically targets?

Reply:
`NET` targets .NET 5.0 and above